dist/index.html

```diff
@@ -19,16 +19,35 @@ <h2 class="user-name-js">welcome, user name</h2>
       <section class="login-display-js login-display hidden">
       </section>
 
-      <section class="customer-dashboard-display-js customer-dashboard-display">
-        <aside class="booking-history">
+      <section class="customer-display-js customer-display">
+        <aside class="booking-history-section">
           <p>my bookings</p>
           <section class="bookings-list-container">past and current bookings</section>
           <p>my total:</p>
         </aside>
-        <section>
-
+        <section class="make-booking-section">
```

Comment:
Need to work on the flex in this region. It's not formatting how I want -> being very stubborn, elements aren't moving for me. A problem for another day though, time to get to some logic.